test/jdk/jdk/jfr/threading/TestDeepVirtualStackTrace.java

```diff
@@ -42,7 +42,7 @@
  * @requires vm.hasJFR & vm.continuations
  * @library /test/lib /test/jdk
  * @modules jdk.jfr/jdk.jfr.internal
- * @run main/othervm -XX:FlightRecorderOptions:stackdepth=2048
+ * @run main/othervm -XX:MaxNewSize=40M -XX:FlightRecorderOptions:stackdepth=2048
```

Comment:
It would be be good to add some comment about why this extra option is necessary.